docs/performance_metrics.md

```diff
@@ -27,3 +29,81 @@ To generate metrics data for selected performance tests, e.g. boot time only:
 ```
 $ ./scripts/dev_cli.sh tests --metrics -- -- --report-file /tmp/metrics.json --test-filter boot_time
 ```
+
+To set timeout or test iterations for all metrics:
+```
+$ ./scripts/dev_cli.sh tests --metrics -- -- --timeout 5 --iterations 10
+```
+
+## Detail Metrics
+
+The following table lists the supported metrics currently:
```

Comment:
Also, it would be good to explain what does `timeout` and `iterations` mean. Say we can add the following here: 

"The `timeout` defines the maximum execution time of each test for each iteration. The `iteration` defines how many times a test needs to be executed to generate the final metrics data. "

Reply:
Agree with you.